src/grasshopper/lib/configuration/gh_configuration.py

```diff
@@ -177,6 +177,16 @@ class ConfigurationConstants:
             "default": 120.0,
             "typecast": typecast_float,
         },
+        "stop_timeout": {
```

Comment:
nit: this name is confusing, it's a graceful shutdown time, correct? can you try a name along those lines?

Reply:
Removed the argument from this PR, need more time to validate it.